lang/python-txamqp/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2009-2015 OpenWrt.org
```

Comment:
It was only recently said that you use this copyright header unless there was a formal legal document transferring copyright from you to OpenWrt.  If that is not the case, I would recommend using your own name in the header.